functions/01-printTime.js

```diff
@@ -2,24 +2,68 @@
  * Write a function printTime
  * 0 hour - midnight
  * 12 hour - noon
- * 53-7 minute - o'clock
+ * 53 - 59 minute - o'clock
+ * 0 - 7 minute - o'clock
  * 8-23 minute - quarter past
  * 24-37 minute - half past
  * 38-52 minute - quarter to
  * @param{number} hour - 0-24 format, hour of a day
  * @param{number} minute - 0-59 format, minute of a day
  * @returns{string} returns a string representation of a time
  */
-function printTime(hour, minute) {
-  return `${hour}:${minute}`;
+ function printTime(hour, minute) {
+  const qualifier = printMinute(minute);
+  const endonym = qualifier === 'quarter to' 
+    || minute >= 53 && minute <= 59 ? 
+    printHour(hour + 1) : printHour(hour);
```

Comment:
- use this logic from `printHour`
https://github.com/zziyad/task/blob/a6e1a499c10ff52bf5d1b9c2eb89e55b80e979cb/functions/01-printTime.js#L36-L37

 - combine with endonym calculation into a separate function
```js
const findEndonym = (hour, minute, qualifier) => .....
// it must calculate an index of ENDONYMS and then return one
```
 - `printHour` function is no more needed and must be removed afterwards